service/version_bundles.go

```diff
@@ -262,11 +262,52 @@ func newVersionBundles() []versionbundle.Bundle {
 				},
 			},
 			Dependencies: []versionbundle.Dependency{},
-			Deprecated:   false,
+			Deprecated:   true,
```

Comment:
When we now create a new version bundle and deprecate an older one there must be a frozen code path and a new one. I do not see the frozen code path but only the new one. This implies any guest cluster running on `2.0.1` will change on reconciliation eventually but changes should only be initialized actively by us or the customer on an explicit update. So if the bug is in the framework itself the broken framework should still be used for `2.0.1` and the fixed framework should be used for `2.0.2`. Otherwise customers will never have an incentive to roll forward in a fast way, which is what we want.